tensorflow/core/ops/io_ops.cc

```diff
@@ -454,6 +455,7 @@ record_bytes: Number of bytes in the record.
 footer_bytes: Number of bytes in the footer, defaults to 0.
 hop_bytes: Number of bytes to hop before each read. Default of 0 means using
         record_bytes.
+encoding: The type of encoding for the file. Defaults to none.
```

Comment:
Might be a good idea to elaborate on what valid values of this attribute can be.